import-export-cli/integration/devFirst_test.go

```diff
@@ -347,6 +347,54 @@ func TestImportAndExportAPIWithDocument(t *testing.T) {
 		testutils.DevFirstUpdatedSampleCaseDestPathSuffix)
 }
 
+//Import Api with a Document and Export that Api with a Document and hidden files
+func TestImportAndExportAPIWithDocumentAndHiddenFiles(t *testing.T) {
+	username := superAdminUser
+	password := superAdminPassword
+	apim := GetDevClient()
+	projectName := base.GenerateRandomName(16)
+
+	args := &testutils.InitTestArgs{
+		CtlUser:   testutils.Credentials{Username: username, Password: password},
+		SrcAPIM:   apim,
+		InitFlag:  projectName,
+		OasFlag:   testutils.TestOpenAPI3DefinitionPath,
+		APIName:   testutils.DevFirstDefaultAPIName,
+		ForceFlag: false,
+	}
+
+	testutils.ValidateInitializeProjectWithOASFlag(t, args)
+
+	projectPath, _ := filepath.Abs(projectName)
+	base.CreateTempDir(t, projectPath+testutils.DevFirstUpdatedSampleCaseDocName)
+	base.CreateTempDir(t, projectPath+testutils.DevFirstUpdatedSampleCaseDocNameWithHiddenFiles)
+
+	//Move doc file to created project
+	srcPathForDoc, _ := filepath.Abs(testutils.DevFirstUpdatedSampleCaseDocPath)
+	destPathForDoc := projectPath + testutils.DevFirstUpdatedSampleCaseDestPathSuffix
+	base.Copy(srcPathForDoc, destPathForDoc)
+
+	//Move docMetaData file to created project
+	srcPathForDocMetadata, _ := filepath.Abs(testutils.DevFirstUpdatedSampleCaseDocMetaDataPath)
+	destPathForDocMetaData := projectPath + testutils.DevFirstUpdatedSampleCaseDestMetaDataPathSuffix
+	base.Copy(srcPathForDocMetadata, destPathForDocMetaData)
+
+	//Move hidden file to created project
```

Comment:
Shall we modify the existing test case to import with a hidden file, without writing a new one? Also make sure that the test to have two document folders, Doc1 and Doc2, where Doc1 consists of non-hidden content (that means a valid document) and Doc2 has .hidden file only.